### ISSUE TITLE
[BUG] DataFrame.at - TypeError: 'method' object is not subcriptable

### ISSUE THREAD

**Issue by miguelusque** [rapidsai/cudf#5329]
**Describe the bug**
Hi,

I am trying to port the following code from Pandas:

```
import pandas as pd
import numpy as np

random_pdf = pd.DataFrame.from_records(np.random.randn(6, 4), columns=list('ABCD'))

index_pdf = pd.Series(['zero', 'one', 'two', 'three', 'four', 'five'])
random_pdf = random_pdf.set_index(index_pdf)

print(random_pdf.at[index_pdf[0], 'A'])
```

to cuDF:

```
import cudf
import cupy

random_gdf = cudf.DataFrame.from_records(cupy.random.randn(6, 4), columns=list('ABCD'))

index_gdf = cudf.Series(['zero', 'one', 'two', 'three', 'four', 'five'])
random_gdf = random_gdf.set_index(index_gdf)

print(random_gdf.at[index_gdf[0], 'A'])
```

but I get the following error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-f9f3b5f56cdc> in <module>
      7 random_gdf = random_gdf.set_index(index_gdf)
      8 
----> 9 print(random_gdf.at[index_gdf[0], 'A'])

TypeError: 'method' object is not subscriptable
```

**Expected behavior**
To return the same value than Pandas

**Environment overview (please complete the following information)**
RAPIDS 0.13 , DGX-1

**Comment by galipremsagar** [comment]
Fixed by #5337

**Comment by miguelusque** [comment]
Fixed in 0.15 nightly. Thanks!!!
